templates/style/style.scss

```diff
@@ -7,18 +7,19 @@
     font-style: normal;
     font-weight: 400;
     src:
-      local('Fira Sans'),
-      url("FiraSans-Regular.woff2") format('woff2'),
-      url("FiraSans-Regular.woff") format('woff');
+        local('Fira Sans'),
```

Comment:
While adding new class, My editor ran `prettier` on this scss file, in order to make it `prettier`. Please let me know if changes are not fine I can just revert the prettier changes :)

Reply:
please revert these, so we can review only the SCSS changes. 

Any formatting should be separate and a separate discussion